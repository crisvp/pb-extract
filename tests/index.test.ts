import { describe, expect, it } from "vitest";

describe("index", () => {
  it("exports objects", async () => {
    const mod = await import("../src/index");

    expect(mod).toBeDefined();
    expect(mod).toMatchSnapshot();
  });
});
