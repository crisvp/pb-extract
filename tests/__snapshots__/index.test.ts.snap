// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`index > exports objects 1`] = `
{
  "addDefaultFields": [Function],
  "extendField": [Function],
  "normalizeDescriptions": [Function],
  "readCollections": [Function],
  "translateRelation": [Function],
  "tsType": [Function],
  "writeDeclarations": [Function],
}
`;
